bin/main

```diff
@@ -1,3 +1,137 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+=begin
+    Start the game
+puts introduction
+puts description
+player enter their name
+player1 choose its symbol
+the symbol is set
+It's the player1 turn
+Loop {
+    display the board
+
+    The player1 enter the position of the case on the board
+    If the player win then the game end or it's dram
+    else
+
+    display the board
+
+    The player 2 enter the position of the case on the board
+    If the player win then the game endor it's dram
+    else
+}
+
+if game end,
+    if player #{player} win, puts player wins
+    if player #{player} loose, puts player loose
+    If draw, puts the game is a draw
+    Replay?
+    
+=end
+
+class Game
+    def initialize
+        introduction
+        @player1 =Player.new(1)
+        @player2 =Player.new(2)
+        @playing=true
+        @player=nil
+        #@board = { 1 => 1, 2 => 2, 3 => 3,4 => 4, 5 => 5, 6 => 6,7 => 7, 8 => 8, 9 => 9}
+        @board = [1,2,3,4,5,6,7,8,9]
+        @boardPlayer = [nil,nil,nil,nil,nil,nil,nil,nil,nil]
+        play(@player1)
+    end
+
+    def introduction
+        puts "Welcome to TIC TAC Toe"
+        puts "Tic-tac-toe is a game for two players, X and O, who take turns marking 
+        the spaces in a 3×3 grid. The player who succeeds in placing 
+        three of their marks in a horizontal, vertical, or diagonal row is the winner."
+    end
+
+    def play(player)
+        puts "Game start"
+        
+        while @playing
+            board      
+            puts " #{player.name} Place your symbol on the board"
+            num = gets.chomp.to_i
+            num = num.to_i
+            num = num- 1
+            @boardPlayer[num]=player.symbol
+        end
```

Comment:
- Game loop that you have implemented is missing the followong:
* [ ] `[REQUIRED]` game tells which player turn it is
* [ ] `[REQUIRED]` game asks player to select from available moves
* [ ] `[REQUIRED]` game informs player if selected move is invalid
* [ ] `[REQUIRED]` game informs player if selected move is a winning move
* [ ] `[REQUIRED]` game informs player if selected move is a draw move

- **[SUGGESTION]** to accomplish the above, you can use conditional methods when checking for the validity of a move or when checking if a move is a winning/draw move. You are allowed to incorporate these methods in your loop as if they were implemented in a different class related to the logic of the game which you will work on in the next milestone.

Reply:
Thank you @stratospherique